xarray/namedarray/_aggregations.py

```diff
@@ -571,7 +571,7 @@ def std(
         dim: Dims = None,
         *,
         skipna: bool | None = None,
-        ddof: int = 0,
+        ddof: int | None = None,
```

Comment:
I know that I'm supposed to regenerate these files instead of changing them directly, but I don't understand why when I change this directly `ddof=0` still gets passed down explicitly.

Reply:
I can't reproduce that. What code do you use to see that behavior? However, if you tried `Dataset.std`, then the reason is that you didn't update the defaults on `DatasetAggregations`, yet.